kolibri/plugins/user_profile/assets/src/routes.js

```diff
@@ -100,28 +103,28 @@ export default [
       {
         path: 'require_account_credentials',
         name: 'REQUIRE_ACCOUNT_CREDENTIALS',
-        component: ToBeDone,
+        component: MergeAccountDialog,
       },
       {
         path: 'admin_password',
         name: 'ADMIN_PASSWORD',
-        component: ToBeDone,
-      },
-      {
-        path: 'show_accounts',
-        name: 'SHOW_ACCOUNTS',
-        component: ToBeDone,
+        component: MergeAccountDialog,
       },
+      // {
+      //   path: 'show_accounts',
```

Comment:
no, we don't need them. I left those lines to remember we had those routes in the original design and now they don't exist. I thought it could be useful in case of doubts during this development but I am deleting them.